apps/entity-service/src/entities/dto/project-pitch.dto.ts

```diff
@@ -0,0 +1,470 @@
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { ApiProperty } from "@nestjs/swagger";
+import { IsArray, IsDate, IsNumber, IsOptional, IsString, IsUUID, Max, Min } from "class-validator";
+import { JsonApiAttributes, pickApiProperties } from "@terramatch-microservices/common/dto/json-api-attributes";
+import { ProjectPitch } from "@terramatch-microservices/database/entities";
+
+@JsonApiDto({ type: "projectPitches" })
+export class ProjectPitchDto extends JsonApiAttributes<ProjectPitchDto> {
+  constructor(data: ProjectPitch) {
+    pickApiProperties(data, ProjectPitchDto);
```

Comment:
This call shouldn't be needed with the call to super below. This is assigning the properties to the DTO object twice.